src/main/java/org/dormitory/autobotsoccub/user/UserPool.java

```diff
@@ -0,0 +1,42 @@
+package org.dormitory.autobotsoccub.user;
+
+import lombok.Getter;
+import net.jodah.expiringmap.ExpirationPolicy;
+import net.jodah.expiringmap.ExpiringMap;
+import org.telegram.telegrambots.api.objects.User;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class UserPool {
+
+    private static final int POOL_SIZE = 4;
+
+    @Getter private Map<Integer, User> userPool = ExpiringMap.builder()
```

Comment:
Do not expose this map to outer world as someone could change its contents. Make explicit getter which returns copy of the map e.g. ImmutableMap.copyOf(userPool)

Reply:
Okay, its done